inc/class-upgrade.php

```diff
@@ -999,8 +999,7 @@ private function upgrade_205() {
 	 * Removes the metadata related to the settings page introduction modal for all the users.
 	 */
 	private function upgrade_207() {
-		global $wpdb;
-		delete_metadata( 'user', 0, '_yoast_settings_introduction', '', true );
+		\add_action( 'shutdown', [ $this, 'delete_user_introduction_meta' ] );
```

Comment:
No need for a backslash since we're not in a namespaced class

```suggestion
		add_action( 'shutdown', [ $this, 'delete_user_introduction_meta' ] );
```